pages/test-evaluate/evaluation-tools/selecting-eval-tools.pl.md

```diff
@@ -0,0 +1,230 @@
+   
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Wprowadzenie" class="full" %}
+{:/}
+
+
+Narzędzia oceny dostępności cyfrowej to programy lub usługi internetowe, które pomagają określić, czy treść internetowa spełnia standardy dostępności. W tym dokumencie omówiono cechy narzędzi oceny i inne aspekty, które należy wziąć pod uwagę przy wyborze narzędzi oceny. Łącza do różnych narzędzi znajdują się na [[Liście narzędzi oceny dostępności cyfrowej]](/test-evaluate/tools/list/). Strona oferuje również filtry, które pomagają dokonać wstępnego wyboru, aby znaleźć najbardziej pomocne narzędzie.
+
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Treść strony" class="full" %}
+{:/}
+
+- TOC jest tworzony automatycznie.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Wideo: Omówienie narzędzi oceny {#video}
+
+{% include video-player-data.html
+    yt-id="bn1XJSjc_qM"
+    video-id="tools-for-evaluating"
+%}
+
+_Ten film jest również dostępny na serwerze W3C: [Wideo: Omówienie narzędzi oceny (format pliku: MP4, rozmiar pliku: 51MB)](https://media.w3.org/wai/evaluation-intros/tools-for-evaluating.mp4)._
+
+
+{% include excol.html type="start" id="video-intro-transcript" %}
+
+Transkrypcja oraz opis wizualnej strony filmu
+
+{% include excol.html type="middle" %}
+
+<table>
+  <thead>
+    <tr>
+      <th width="50%">Dźwięk</th>
+      <th>Obraz</th>
+    </tr>
+  </thead>
+<tbody>
+  <tr>
+    <td>Narzędzia oceny dostępności cyfrowej</td>
+    <td>Narzędzia oceny dostępności cyfrowej</td>
+  </tr>
+  <tr>
+    <td>Istnieją programy i usługi internetowe, które pomogą Ci rozpoznać problemy z dostępnością.</td>
+    <td>Otwiera się skrzynka narzędziowa przy komputerze. Ze skrzynki wychodzi lupa, aby sprawdzić stronę internetową na komputerze. Lupa powiększa słowo „dostępność”.</td>
+  </tr>
+  <tr>
+    <td>Mogą one zaoszczędzić wiele czasu i wysiłku podczas oceny oraz zapobiec powstawaniu nowych barier dostępności.</td>
+    <td>Oszczędzaj czas i wysiłek. Unikaj barier dostępności.</td>
+  </tr>
+  <tr>
+    <td>Jednak narzędzia nie mogą zrobić wszystkiego. Niektórych testów dostępności po prostu nie można zautomatyzować i muszą być przeprowadzone ręcznie.</td>
+    <td>Osoba obok komputera ze stroną internetową używa lupy ze słowem „dostępność”.</td>
+  </tr>
+  <tr>
+    <td>Niektóre narzędzia poprowadzą Cię przez te testy, których nie można zautomatyzować.</td>
+    <td>Lista kontrolna obok komputera.</td>
+  </tr>
+  <tr>
+    <td>Niektóre narzędzia sprawdzają tylko jedną stronę na raz, inne mogą sprawdzać całe witryny.</td>
+    <td>Strona w witrynie internetowej jest skanowana. Nad elementami strony pojawia się kolejno znak błędu, powodzenia i znak zapytania.</td>
```

Comment:
@zwiastunsw The description seems to have been reworded. Would you think of an alternative to get closer to the original wording?